src/GPE.jl

```diff
@@ -160,45 +145,38 @@ Initialise the marginal log-likelihood of Gaussian process `gp`.
 function initialise_mll!(gp::GPE{X,Y,M,K,P,D}) where {X,Y,M,K,P,D}
     μ = mean(gp.mean,gp.x)
     Σ = cov(gp.kernel, gp.x, gp.data)
-    gp.cK = P.name.wrapper(Σ + (exp(2*gp.logNoise) + 1e-5)*I)
+    gp.cK = init_cK(P, Σ + (exp(2*gp.logNoise) + 1e-5)*I)
     y = gp.y - μ
     gp.alpha = gp.cK \ y
     # Marginal log-likelihood
     gp.mll = -(dot(y, gp.alpha) + logdet(gp.cK) + log2π * gp.nobs) / 2
     gp
 end
+init_cK(::Type{<:PDMat}, m) = PDMat(m)
```

Comment:
The default could apply to all AbstractPDMat, not just PDMat specifically.

Reply:
Using the wrapper? I.e. 
```julia
init_cK(P::Type{<:AbstractPDMat}, m) = P.name.wrapper(m)
```
or in the form of the changes proposed above 
```julia 
init_cK(gp::GPE{X,Y,M,K,P,D}) where {X,Y,M,K,P,D} = P.name.wrapper(Σ_default(gp))
``` 
which would merge the two `init_cK` lines.
However, I thought, you wanted to get rid of this, and I think it makes sense, because this is most likely not in the "public" API of Julia.

Reply:
Yes, I agree, there doesn't seem to be an easy way to instantiate PDMats. You would think `convert(P, Σ_default(gp))` would work, but it doesn't.